05/app.js

```diff
@@ -1,3 +1,21 @@
-console.log('DOM');
-
 const curr = document.querySelector('.js-curr');
+const secondBtn = curr.cloneNode(true);
+
+curr.parentElement.appendChild(secondBtn);
+secondBtn.innerText = 'Usuń z koszyka';
+
+for (let i=0; i<curr.parentElement.children.length; i++) { //co wyskakuje gdy i<=? widziałem jeden element więcej i ostatni jest undefined
```

Comment:
Bo wychodisz poza zakres tablicy np. `const arr = ['a', 'b', 'c']` => `arr[3]` => `undefined` bo istniejące indeksy to 0, 1, 2 czyli o jeden mniej niż elementów w tablic tj. `arr.length -1`  czyli nie może być równości